domains/client/eth-service/src/rpc.rs

```diff
@@ -86,14 +89,24 @@ impl<Client, TxPool, CA: ChainApi, CT: Clone, Block: BlockT, BE> Clone
             fee_history_cache_limit: self.fee_history_cache_limit,
             execute_gas_limit_multiplier: self.execute_gas_limit_multiplier,
             forced_parent_hashes: self.forced_parent_hashes.clone(),
+            pending_inherent_data_provider: self.pending_inherent_data_provider.clone(),
         }
     }
 }
 
 /// Instantiate Ethereum-compatible RPC extensions.
-pub(crate) fn create_eth_rpc<Client, BE, TxPool, CA, CT, Block, EC: EthConfig<Block, Client>>(
+pub(crate) fn create_eth_rpc<
+    Client,
+    BE,
+    TxPool,
+    CA,
+    CT,
+    Block,
+    EC: EthConfig<Block, Client>,
```

Comment:
I prefer to always use `where`, that way it is more readable, especially when number of generics increases